src/main/resources/bullet_kafka_defaults.yaml

```diff
@@ -1,33 +1,65 @@
 # These are required Kafka settings for consumers and producers
```

Comment:
Nit: Add a period for each sentence in your comments.

Reply:
I added periods to sentences. If it's a title, it looks weird, so I left it out.